crates/ekuad/src/lib.rs

```diff
@@ -5,14 +5,22 @@ use ekore;
 #[no_mangle]
 pub unsafe extern "C" fn quad_ker(x: f64, extra: *mut c_void) -> f64 {
     let ex = *(extra as *mut Extra);
-    ekore::ciao(x * ex.slope, ex.shift)
+    //ekore::ciao(x * ex.slope, ex.shift)
+    (ex.py)(x * ex.slope, ex.shift)
```

Comment:
Perfect, thanks @felixhekhorn 

Here also something like `ekore::ciao` had a role, because it was showing that it could make use of functions defined in siblings' crates.

However, the composition is trivial, once they are all available functions, so I would not bother complicating the example, but at this point rather aim for the actual thing.